### PR TITLE
feature: add env flag for enabling download

### PR DESCRIPTION
**Pull request by skeptrunedev**

### CODE REVIEW

File: server/src/handlers/file_handler.rs [hunk 1]
```diff
@@ -158,6 +158,13 @@ pub async fn get_file_handler(
     user: Option<LoggedUser>,
     _required_user: RequireAuth,
 ) -> Result<HttpResponse, actix_web::Error> {
+    let download_enabled = std::env::var("DOCUMENT_DOWNLOAD_FEATURE").unwrap_or("off".to_string());
```
Comment (densumesh) [inline server/src/handlers/file_handler.rs:161]: Don't we want to default to on?

Comment (skeptrunedev) [reply server/src/handlers/file_handler.rs:161]: Hmmm, yeah. Let me change it. Conflicted bc Match was concerned from a security perspective

Comment (skeptrunedev) [reply server/src/handlers/file_handler.rs:161]: Eh, changed my mind again. Leaving default off to reduce the odds of them messing up.

**Review by densumesh — APPROVED** [review]
LGTM

**Pull request merged**